fetch_issues.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
a script that can fetch all issues from a JQL query.
Exporting 11,000 issues for jenkins core via web-ui wasn't very fun especially given how slow the export was so I scripted it, works nicely